runtime/src/dispatcher.rs

```diff
@@ -45,11 +47,11 @@ use crate::{
         types::TxnBatch,
         Context as TxnContext,
     },
-    types::{Body, ComputedBatch, HostStorageEndpoint},
+    types::{Body, ComputedBatch, Error, HostStorageEndpoint},
 };
 
 /// Maximum amount of requests that can be in the dispatcher queue.
-const BACKLOG_SIZE: usize = 10;
+const BACKLOG_SIZE: usize = 1000;
```

Comment:
could you add a summary of why there are so many changes to dispatcher.rs

Reply:
Some minor refactoring to avoid repetition with sending responses to various requests back. Previously each handler had its own `send_response` but there's really no need for that.

Reply:
That is just some minor refactoring to avoid repeating sending back responses from handlers. Previously each handler had its own `send_response` but there's really no need for that.

Reply:
ah thanks